File: C/Listas/Lista 02/ex01.c
//Faça um programa que leia a idade de uma pessoa expressa em anos, meses e dias e mostre-a expressa apenas em dias.
#include <stdio.h>

int main(){
    int  idadetotal, anos, meses, dias;
    printf("Insira, respectivamente, o total de anos, meses e dias de vida:\n");
    scanf("%d%d%d", &anos, &meses, &dias);
    idadetotal=((anos*12)+meses)*30+dias;
    printf("Você viveu cerca de %d dias!", idadetotal);
    return 0;
}

File: Java/extra/Lista 02/src/ex01.java
//Faça um programa que leia a idade de uma pessoa expressa em anos, meses e dias e mostre-a expressa apenas em dias. 

import java.util.Scanner;

public class ex01 {

    static int total_dias(int a, int b, int c) {
        int x = (a * 365) + (b * 30) + c;
        return x;
    }

    public static void main(String[] args) {
        int anos, meses, dias;
        try (Scanner leitor = new Scanner(System.in)) {
            System.out.println("Digite sua idade em anos: ");
            anos = leitor.nextInt();
            System.out.println("\nDigite os meses vividos desde que completou " + anos + " anos: ");
            meses = leitor.nextInt();
            System.out.println("\nInforme os dias vividos desde que completou " + anos + " e " + meses + " meses: ");
            dias = leitor.nextInt();
        }
        System.out.printf("\nVocê viveu um total de %d dias!", total_dias(anos, meses, dias));
    }

}